tests/suites/test_suite_pkparse.data

```diff
@@ -915,19 +915,19 @@ depends_on:MBEDTLS_AES_C:MBEDTLS_MD_CAN_SHA384:MBEDTLS_PKCS5_C:MBEDTLS_CIPHER_C:
 pk_parse_keyfile_rsa:"data_files/rsa_pkcs8_pbes2_pbkdf2_2048_aes256cbc_sha384.der":"PolarSSLTest":0
 
 Parse RSA Key #100.1 (512-bit)
-depends_on:MBEDTLS_PEM_C
+depends_on:MBEDTLS_PEM_PARSE_C
```

Comment:
The problem is that some RSA algorithms involve a hash and the key must be larger than some function of the hash. A 512-bit key is ok with SHA-256, but not when the smallest hash is SHA-384 or SHA-512. There are several possible solutions here:

* During exercising, if the key is RSA and the hash is too big compared with the key for the given algorithm, expect `PSA_ERROR_INVALID_ARGUMENT` instead of `PSA_SUCCESS`.
  I'm not keen on adding exceptions inside exercising because those exceptions might fire up in unintended contexts, so we'd end up with what we think are positive test cases, but actually the library is erroring in some unexpected way.
* During exercising, if the key is RSA and the hash is too big compared with the key for the given algorithm, skip that algorithm.
  As above, I'm not keen on this.
* In that specific test (which is RSA-only), for each algorithm, if the hash that would be used for exercising is too big compared with the key, skip exercising with that algorithm.
  That's flexible and not dangerous, but adds a lot of non-orthogonality.
* Replace the test data with keys that are large enough for all supported hashes. We don't have a test objective here of testing with keys of any particular order of magnitude, the only reason we're testing with small keys is that it's faster.
  This has my preference so far.